<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_serv_xindianjiaozhun</class>
 <widget class="QDialog" name="Dlg_serv_xindianjiaozhun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>心电校准</string>
  </property>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>111</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>心电校准</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_jiaozhun">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
